src/renderer/data/accounts/getAccount.ts

```diff
@@ -0,0 +1,121 @@
+import * as sol from '@solana/web3.js';
+import hexdump from 'hexdump-nodejs';
+// import randomart from 'randomart'
+import { LRUCache } from 'typescript-lru-cache';
+
+import { AccountInfo } from './accountsSlice';
+
+import { Net, netToURL } from '../ValidatorNetwork/validatornetworkSlice';
+
+export const BASE58_PUBKEY_REGEX = /^[1-9A-HJ-NP-Za-km-z]{32,44}$/;
+
+const HEXDUMP_BYTES = 512;
+
+// TODO: make a gloabal state of "current slot number"
```

Comment:
i need this for transaction following (which I think I want to make replace program changes) as well